app/javascript/components/custom-url-tabs/index.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Tab, Tabs, Modal } from 'carbon-components-react';
+import { checkForFormChanges } from './helper';
+
+const CustomURLTabs = ({
+  tabs, path, currentTab, checkForChanges,
+}) => {
+  const [{ selectedTab, showConfirm, url }, setState] = useState({ selectedTab: 0, showConfirm: false });
+  const activeTabClassName = 'bx--tabs--scrollable__nav-item--selected';
+
+  const onTabClick = (id) => {
+    if (currentTab !== id) {
+      if (checkForChanges && checkForFormChanges()) {
+        setState((state) => ({
+          ...state,
+          showConfirm: true,
+          url: `${path}${id}?uib-tab=${id}`,
+        }));
+        document.getElementById(id).parentElement.classList.add(activeTabClassName);
+      } else {
+        window.location.replace(`${path}${id}?uib-tab=${id}`);
+      }
+    }
+  };
+
+  const fixTabStyling = () => {
+    const selectedTabs = [];
+    document.getElementsByClassName(activeTabClassName).forEach((element) => {
+      selectedTabs.push(element);
+    });
+    selectedTabs.forEach((tab) => {
+      tab.classList.remove(activeTabClassName);
+    });
```

Comment:
could you try to replace this to see if it works?
```
document.getElementsByClassName(activeTabClassName).forEach((element) => {
      element.classList.remove(activeTabClassName);
    });
```

Reply:
It works, I pushed all the changes if you can review again @jeffibm

Reply:
Hey @GilbertCherrie , I was trying to remove the array `selectedTab`. I am still seeing the same thing and its not mentioned `outdated`.

Reply:
Not sure why it didn't push properly, it should be good now if you can review again